src/stylesheets/settings/_settings-color.scss

```diff
@@ -18,57 +18,57 @@ Theme palette colors
+$theme-color-secondary-family:      'red' !default;
+$theme-color-secondary-lightest:    false !default;
+$theme-color-secondary-lighter:     '#{$theme-color-secondary-family}-10' !default;
+$theme-color-secondary-light:       '#{$theme-color-secondary-family}-30' !default;
+$theme-color-secondary:             '#{$theme-color-secondary-family}-50' !default;
+$theme-color-secondary-vivid:       '#{$theme-color-secondary-family}-50v' !default;
+$theme-color-secondary-dark:        '#{$theme-color-secondary-family}-60v' !default;
+$theme-color-secondary-darker:      '#{$theme-color-secondary-family}-70v' !default;
+$theme-color-secondary-darkest:     false;
 
 // Accent warm colors
-$theme-color-accent-warm-family:   'orange' !default;
-$theme-color-accent-warm-lightest: false !default;
-$theme-color-accent-warm-lighter:  $theme-color-accent-warm-family, 10 !default;
-$theme-color-accent-warm-light:    $theme-color-accent-warm-family, 20, vivid !default;
-$theme-color-accent-warm:          $theme-color-accent-warm-family, 30, vivid !default;
-$theme-color-accent-warm-dark:     $theme-color-accent-warm-family, 50, vivid !default;
-$theme-color-accent-warm-darker:   $theme-color-accent-warm-family, 60 !default;
-$theme-color-accent-warm-darkest:  false !default;
+$theme-color-accent-warm-family:    'orange' !default;
+$theme-color-accent-warm-lightest:  false !default;
+$theme-color-accent-warm-lighter:   '#{$theme-color-accent-warm-family}-10' !default;
+$theme-color-accent-warm-light:     '#{$theme-color-accent-warm-family}-20v' !default;
+$theme-color-accent-warm:           '#{$theme-color-accent-warm-family}-30v' !default;
+$theme-color-accent-warm-dark:      '#{$theme-color-accent-warm-family}-50v' !default;
+$theme-color-accent-warm-darker:    '#{$theme-color-accent-warm-family}-60' !default;
+$theme-color-accent-warm-darkest:   false !default;
 
 // Accent cool colors
-$theme-color-accent-cool-family:   'cyan' !default;
-$theme-color-accent-cool-lightest: false !default;
-$theme-color-accent-cool-lighter:  $theme-color-accent-cool-family, 5 !default;
-$theme-color-accent-cool-light:    $theme-color-accent-cool-family, 20 !default;
-$theme-color-accent-cool:          $theme-color-accent-cool-family, 30, vivid !default;
-$theme-color-accent-cool-dark:     $theme-color-accent-cool-family, 40, vivid !default;
-$theme-color-accent-cool-darker:   'blue', 60 !default;
-$theme-color-accent-cool-darkest:  false !default;
+$theme-color-accent-cool-family:    'cyan' !default;
+$theme-color-accent-cool-lightest:  false !default;
+$theme-color-accent-cool-lighter:   '#{$theme-color-accent-cool-family}-5' !default;
+$theme-color-accent-cool-light:     '#{$theme-color-accent-cool-family}-20' !default;
+$theme-color-accent-cool:           '#{$theme-color-accent-cool-family}-30v' !default;
+$theme-color-accent-cool-dark:      '#{$theme-color-accent-cool-family}-40v' !default;
+$theme-color-accent-cool-darker:    'blue-60' !default;
```

Comment:
Same as above: should this be `'#{$theme-color-accent-cool-family}-60v'` and overridden in the theme file?